ui/src/shared/constants/fluxFunctions.ts

```diff
@@ -513,9 +581,9 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
         type: 'Integer',
       },
       {
-        name: 'columns',
-        desc: 'List of columns by which to sort. Default is `["_value"]`.',
-        type: 'Array of Strings',
+        name: 'column',
+        desc: 'Column by which to sort. Default is `"_value"`.',
```

Comment:
spec still has columns: https://github.com/influxdata/flux/blob/master/docs/SPEC.md#highestlowest

Reply:
Looks like the SPEC needs to be updated here. This was updated with all the aggregate functions so they only accept a single column now:
https://github.com/influxdata/flux/blob/928e70566bea61accd45c43ec04b1692f740964e/stdlib/universe/universe.flux#L171